internal/serverinstall/nomad.go

```diff
@@ -133,6 +139,39 @@ func (i *NomadInstaller) Install(
 		}, nil
 	}
 
+	if strings.ToLower(i.config.volumeType) == "csi" {
+		if i.config.csiVolumeProvider == "" {
+			return nil, fmt.Errorf("please include '-nomad-csi-volume-provider' flag")
+		}
+
+		s.Update("Creating persistent volume")
+
+		vol := api.CSIVolume{
+			ID:   "waypoint",
+			Name: "waypoint",
+			RequestedCapabilities: []*api.CSIVolumeCapability{
+				{
+					AccessMode:     "single-node-writer",
+					AttachmentMode: "file-system",
+				},
+			},
+			MountOptions: &api.CSIMountOptions{
+				FSType:     "xfs",
+				MountFlags: []string{"noatime"},
+			},
+			RequestedCapacityMin: 1073741824,
+			RequestedCapacityMax: 2147483648,
```

Comment:
It seems like these should be configurable too? We should also abstract them out as defaults and include a comment on the size in MB or GB